Trimble.Modus.Components/Controls/Button/TMButton.cs

```diff
@@ -151,6 +85,11 @@ public ImageSource IconSource
             get => (ImageSource)GetValue(IconSourceProperty);
             set => SetValue(IconSourceProperty, value);
         }
+        public ImagePosition IconPosition
```

Comment:
According to this, the user can have the icon in either left or right. But a modus button can have both left and right icon. So we need LeftIconSource and RightIconSource properties.

<img width="217" alt="Screen Shot 2023-06-01 at 2 51 05 PM" src="https://github.com/trimble-oss/modus-mobile-maui-components/assets/60956922/5a1f9c6a-a510-4423-928d-0a71d8feeaf5">